app/models/concerns/radbear_user.rb

```
@@ -191,6 +191,10 @@ def test_sms!(from_user)
     UserSMSSenderJob.perform_later 'Test SMS', from_user.id, id, nil, false
   end
 
+  def reactivate
+    update(last_activity_at: nil)
```

use `update!` so this can't silently fail

---

nevermind :)